site/app/libraries/database/DatabaseQueries.php

```diff
@@ -5133,8 +5130,6 @@ public function createGradeable(\app\models\gradeable\Gradeable $gradeable) {
               g_ta_view_start_date,
               g_grade_start_date,
               g_grade_due_date,
-              g_grade_released_date,
```

Comment:
we probably want to keep this line?

Reply:
Yes. Not sure why that got removed but will fix it.